samples/graph-app-installation-lifecycle/nodejs/.env

```diff
@@ -1,3 +1,5 @@
-ClientId =
-ClientSecret =
-AllowedHosts = *
\ No newline at end of file
+ClientId=
+ClientSecret=
+AllowedHosts=*
+BROWSER=none
```

Comment:
It is not a React app, why do we need `HTTPS` and `BROWSER`?

Reply:
BROWSER is removed.
HTTPS - In local it was running HTTP, But in TTK ENDPOINT is running only through HTTPS and so we added.